core/services/pipeline/runner.go

```diff
@@ -583,8 +583,10 @@ func (r *runner) runReaper() {
 
 	err := r.orm.DeleteRunsOlderThan(ctx, r.config.JobPipelineReaperThreshold())
 	if ctx.Err() != nil {
+		r.lggr.Errorw("Pipeline run reaper timed out", "error", err)
```

Comment:
Is this necessarily the case, that timed out? Could it have been cancelled? Would it be simpler to just drop this if block and handle the error normally below? It will say what happened already.

Reply:
agreed.